### PR TITLE
Add installation instructions using ponyup

### PR DESCRIPTION
**Pull request by ordepdev**
Fixes https://github.com/ponylang/corral/issues/45.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,14 @@ Pony dependency management command-line tool.
 
 Corral is alpha level software. We advise you use [stable](https://github.com/ponylang/pony-stable) for your dependency management needs. We are looking for help building out Corral and would love if you help out with any of the [open issues](https://github.com/ponylang/corral/issues).
 
+## Instalation
+
+The following command will download the latest release of `corral` and install it to `~/.pony/ponyup/bin` by default.
```
Comment (Theodus) [inline README.md]: This should probably include a link to the ponyup repo, in case they don't already have that installed.

Comment (ordepdev) [reply README.md]: Improved a bit, let me know what you think.

File: README.md [hunk 2]
```diff
@@ -8,6 +8,14 @@ Pony dependency management command-line tool.
 
 Corral is alpha level software. We advise you use [stable](https://github.com/ponylang/pony-stable) for your dependency management needs. We are looking for help building out Corral and would love if you help out with any of the [open issues](https://github.com/ponylang/corral/issues).
 
+## Instalation
```
Comment (SeanTAllen) [inline README.md]: This should be `Installation`

Comment (ordepdev) [reply README.md]: My bad.

**Comment by ordepdev** [comment]
Just moved the section after the about one.

**Comment by SeanTAllen** [comment]
@Theodus it's ok to merge this now, yes?

**Comment by Theodus** [comment]
Yes

**Comment by SeanTAllen** [comment]
Thanks, @ordepdev.

**Pull request merged**